CleverRaven/Cataclysm-DDA
# 8 new UI colour themes

#### Summary
Interface "8 new UI colour themes"


#### Purpose of change



After an afternoon where I had a lot of screen glare and didn't find a light theme that was to my liking, I decided to make a theme that would resolve this issue. While I was at it, I got hooked and ended up making a few more stylistic colour themes since most of the themes currently available feel a bit same-y. The purpose is to give the user some more options on how to experience the UI.

#### Describe the solution



From a technical aspect, it is very simple. I simply created 8 new Json files with the colour definitions.  These files are added to data > raw > color_themes directory. that's it.

No existing code is changed as this is purely additive. 

#### Describe alternatives you've considered



none, as this seemed the way the game handles the colour schemes.

#### Testing



Testing is done as follows:
1. add the files to the cdda > data > raw > color_themes directory. 
2. start the game
3. on the main screen, go to settings  > colour > colour themes and select one of the new themes. 
    The shortcuts to reach the theme menu are: t > r > c. using the shortcuts speeds up testing.
4. The colour theme doesn't take effect immediately, so close and restart the game. you should now already see the theme in effect.
5. For further testing, I made sure to check out the map, side panel, log, crafting panel and inventory with the new themes for readability and style. 

#### Additional context


Spark:
![Theme demonstration - Spark](https://github.com/CleverRaven/Cataclysm-DDA/assets/145038845/bcbf0148-debd-4ac8-879c-1663e454383c)

Vector:
![Theme demonstration - Vector](https://github.com/CleverRaven/Cataclysm-DDA/assets/145038845/40b828e9-d6b9-4dbf-bcd9-581b5b117947)

Sun:
![Theme demonstration - Sun](https://github.com/CleverRaven/Cataclysm-DDA/assets/145038845/64949c19-5a88-4dde-9a2e-377038d6518c)

Shogun:
![Theme demonstration - Shogun](https://github.com/CleverRaven/Cataclysm-DDA/assets/145038845/80d6550e-d39b-4d73-9eb1-b13d0548c89d)

Oxygen:
![Theme demonstration - Oxygen](https://github.com/CleverRaven/Cataclysm-DDA/assets/145038845/d8ac8ef7-b5ed-40e2-a932-2d8b7d589088)

Empyrium:
![Theme demonstration - Empyrium](https://github.com/CleverRaven/Cataclysm-DDA/assets/145038845/e3fdd3c7-b6bc-4ab5-bec7-3da8b71e55e5)

Blood Moon:
![Theme demonstration - Blood Moon](https://github.com/CleverRaven/Cataclysm-DDA/assets/145038845/b847db51-09b9-4d3b-9383-fac0516baf32)

Abyss:
![Theme demonstration - Abyss](https://github.com/CleverRaven/Cataclysm-DDA/assets/145038845/837540de-db25-41f8-97cb-f4fd8adddc93)
